commercetools-models/src/main/java/io/sphere/sdk/carts/CartsConfiguration.java

```diff
@@ -6,9 +6,13 @@
 public interface CartsConfiguration {
     
     Integer getDeleteDaysAfterLastModification();
+    Boolean getCountryTaxRateFallbackEnabled();
 
     static CartsConfiguration of(final Integer deleteDaysAfterLastModification) {
         return CartsConfigurationImpl.of(deleteDaysAfterLastModification);
     }
-    
+
+    static CartsConfiguration of(final Boolean countryTaxRateFallbackEnabled) {
```

Comment:
Either you should also a `of(final Integer deleteDaysAfterLastModification, final Boolean countryTaxRateFallbackEnabled)` method

or just remove it as there is a specific update action for this flag available and both parameters are optional.